frontend/src/metabase/admin/datamodel/components/database/MetadataTablePicker.jsx

```diff
@@ -63,14 +65,28 @@ class MetadataTablePicker extends Component {
   }
 }
 
-export default Tables.loadList({
-  query: (state, { databaseId }) => ({
-    dbId: databaseId,
-    include_hidden: true,
-    ...PLUGIN_FEATURE_LEVEL_PERMISSIONS.dataModelQueryProps,
+export default _.compose(
+  Databases.load({
+    id: (state, { databaseId }) =>
+      databaseId !== SAVED_QUESTIONS_VIRTUAL_DB_ID ? databaseId : undefined,
   }),
-  reloadInterval: (state, props, tables = []) => {
-    return tables.some(t => isSyncInProgress(t)) ? RELOAD_INTERVAL : 0;
-  },
-  selectorName: "getListUnfiltered",
-})(MetadataTablePicker);
+  Tables.loadList({
+    query: (state, { databaseId }) => ({
+      dbId: databaseId,
+      include_hidden: true,
+      ...PLUGIN_FEATURE_LEVEL_PERMISSIONS.dataModelQueryProps,
+    }),
+    reloadInterval: (state, { database }, tables = []) => {
+      if (
+        database &&
+        isSyncInProgress(database) &&
+        tables.some(t => isSyncInProgress(t))
+      ) {
+        return RELOAD_INTERVAL;
+      } else {
+        return 0;
```

Comment:
Using zero as no reloading makes readers doubt it and check the code of EntityLoaders, maybe we can use null for that